src/App.tsx

```diff
@@ -14,11 +14,100 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[];
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: goodsFromServer,
+    selectedGoods: ['Jam'],
+  };
+
+  addWord = (word: string) => {
+    this.setState((prevState) => ({
+      selectedGoods: [...prevState.selectedGoods, word],
+    }));
+  };
+
+  removeWord = (word: string) => {
+    this.state.selectedGoods.splice(
+      this.state.selectedGoods.indexOf(word), 1,
+    );
+
+    this.setState((prevState) => ({
+      selectedGoods: prevState.selectedGoods,
+    }));
+  };
+
+  resetList = () => {
+    this.setState({ selectedGoods: [] });
+  };
+
+  render() {
+    let message = '';
+
+    if (this.state.selectedGoods.length === 0) {
+      message = 'No goods selected';
+    }
+
+    if (this.state.selectedGoods.length === 1) {
+      message = '- is selected';
+    }
+
+    if (this.state.selectedGoods.length > 1) {
+      message = '- are selected';
+    }
+
+    return (
+      <div className="App">
+        <div className="title">
+          <h1 className="title__article">
+            {this.state.selectedGoods[this.state.selectedGoods.length - 1]}
+            {' '}
+            {message}
+          </h1>
+
+          <button
+            type="button"
+            className="title__button"
+            onClick={() => {
+              this.resetList();
+            }}
+          >
```

Comment:
If the function is short it is not necessary create a method, try write like this:
```suggestion
            className="title__button"
            onClick={() => {
                this.setState({ selectedGoods: [] });
              }}
           >
```